app/main.py

```diff
@@ -1 +1,4 @@
-# write your code here
+def copy_file(command):
```

Comment:
This code looks like magic to an outsider)
Could you create intermediate variables to make it more readable)